contracts/liquidity_hub/epoch-manager/src/state.rs

```diff
@@ -1,8 +1,8 @@
 use cw_controllers::{Admin, Hooks};
 use cw_storage_plus::{Item, Map};
-use white_whale_std::epoch_manager::epoch_manager::{Config, EpochV2};
+use white_whale_std::epoch_manager::epoch_manager::{Config, Epoch};
 
 pub const CONFIG: Item<Config> = Item::new("config");
 pub const ADMIN: Admin = Admin::new("admin");
 pub const HOOKS: Hooks = Hooks::new("hooks");
-pub const EPOCHS: Map<&[u8], EpochV2> = Map::new("epochs");
+pub const EPOCHS: Map<&[u8], Epoch> = Map::new("epochs");
```

Comment:
Yes rename it to Epoch. We used EpochV2 before because the struct was in the same place where the other Epoch struct was defined, but now it's moved to another place so use that one.